meetings/2024-08-28.md

```diff
@@ -0,0 +1,78 @@
+# Node.js  TypeScript team Meeting 2024-08-28
+
+## Links
+
+* **Recording**: <https://zoom.us/rec/share/I0O7u3h6xFIay4lOBrIuvOXJNyDQloSdxk3IOmCjU-iaR_ExDTlcsQjAW_VMOpTf.BButid5V-V6h1-oq>
+* **GitHub Issue**: <https://github.com/nodejs/typescript/issues/5>
+
+## Present
+
+* Typescript team: @nodejs/typescript
+* Marco Ippolito: @marco-ippolito
+* Rob Palmer @robpalme
+* Lucas Santos @khaosdoctor
+* Andrew Branch @andrewbranch
+* Daniel Rosenwasser @DanielRosenwasser
+* Michael Dawson @mhdawson
+* ZiJian Liu @Lxxyx
+* Shadrach Ekpo @shadrach-se
+* Jake Bailey @jakebailey
+* Jake Yuesong Li @jakecastelli
+
+## Agenda
+
+## Announcements
+
+* Extracted from **typescript-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/loaders
+
+* Roadmap for experimental TypeScript support [#217](https://github.com/nodejs/loaders/issues/217)
+
+  * Node.js advancing in the roadmap, fixing bugs, (some work on sourcemaps).
+  * Consensus is before unflagging we need to test with bigger codebase and have proper tooling.
+  * TypeScript working on <https://github.com/microsoft/TypeScript/issues/59601>
+to support –strip-types with ‘.ts’ extensions (might be released on ts 5.7).
+  * Andrew showcasing the work.
+    * showed a way to use the Node.js experimental features with monorepos
```

Comment:
Is there a link to this prototype?

Reply:
tsc work: https://github.com/microsoft/TypeScript/pull/59767
in use: https://github.com/arethetypeswrong/arethetypeswrong.github.io/pull/194